regolith/main.py

```diff
@@ -257,6 +257,20 @@ def create_parser():
     return p
 
 
+def merge_nested_dicts(dict0: dict, dict1: dict):
```

Comment:
I wonder if this should be imported from ``tools.py``?

Reply:
I put it in tools.